containers/images/foreman-proxy/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM centos:7
+
+RUN echo "tsflags=nodocs" >> /etc/yum.conf && \
+    yum -y install epel-release && \
+    yum -y install http://yum.theforeman.org/releases/nightly/el7/x86_64/foreman-release.rpm && \
+    yum -y install foreman-release-scl && \
+    yum -y install foreman-proxy && \
+    yum -y install rubygem-smart_proxy_pulp rubygem-smart_proxy_remote_execution_ssh rubygem-smart_proxy_discovery rubygem-smart_proxy_ansible rubygem-smart_proxy_openscap && \
+    yum clean all
+
+COPY container-assets/logs.yml /etc/foreman-proxy/settings.d/logs.yml
+COPY container-assets/pulp.yml /etc/foreman-proxy/settings.d/pulp.yml
+COPY container-assets/puppet.yml /etc/foreman-proxy/settings.d/puppet.yml
+COPY container-assets/puppet_proxy_puppet_api.yml /etc/foreman-proxy/settings.d/puppet_proxy_puppet_api.yml
+COPY container-assets/settings.yml /etc/foreman-proxy/settings.yml
+
+RUN ln -s /etc/foreman-proxy/settings.d /usr/share/foreman-proxy/config/settings.d
+
+CMD ["/usr/share/foreman-proxy/bin/smart-proxy"]
```

Comment:
Why not using `ENTRYPOINT` here?

Reply:
I thought as I understood things, command should be what your container actually runs and it is more easily overriden if we need to for example just get inside the container for inspection? I find them both kinda nuanced to be honest.

Reply:
> it is more easily overriden if we need to for example just get inside the container for inspection

IIRC to get inside a container, `docker exec` is used and from my experience it doesn't use `ENTRYPOINT` or `CMD`.

Although, there is a slight difference between them, it does change the user experience.
So I guess if `smart-proxy` doesn't get arguments then `CMD` is good enough.